experimental/fRIP/fRIP_MPRNA_LibraryWorkupProtocol.md

```diff
@@ -0,0 +1,115 @@
+**Protocol typed by Evan Hass**  
+**Last modified on 2020-05-05**
+
+# DNase treatment of RNA  
+1. Aliquot 200 ng of RNA from each sample into PCR tubes, and bring to 16 uL with nuclease-free water. If a given sample is too dilute for 16 uL to contain 200 ng of RNA, just use 16 uL.  
+2. Make DNase master mix (1x recipe):  
+
+- 2 uL 10x Worthington DNase I Buffer  
+- 2 uL Worthington DNase I (Fisher NC9878502)  
+
+3. Add 4 uL of DNase master mix to each PCR tube  
+4. Incubate reactions at 37°C for 30 minutes  
+5. Stop reactions by incubating at 75°C for 10 minutes  
+
+# cDNA Synthesis  
+**_NOTE: All reagents for this cDNA synthesis protocol are included in the SuperScript IV First-Strand Synthesis System (Thermo Fisher 18091200)._**  
+
+1. Transfer 10 uL of DNase-treated RNA to new PCR tubes, and save the other 10 uL  
+2. Make priming master mix (1x recipe):  
+
+- 1 uL nuclease-free water  
+- 1 uL 50 ng/uL Random Hexamers  
+- 1 uL 10 mM dNTP mix  
+
+3. Add 3 uL priming master mix to each tube  
+4. Incubate at 65°C for 5 minutes, then snap-cool on ice  
+5. Make RT master mix:  
+
+- 4 uL 5x SuperScript IV Buffer  
+- 1 uL 100 mM DTT  
+- 1 uL RNaseOUT  
+- 1 uL SuperScript IV  
+
+6. Add 7 uL of RT master mix to each PCR tube  
+7. Run the following program in the thermal cycler:  
+
+- 23°C for 10 minutes  
+- 55°C for 10 minutes  
+- 80°C for 10 minutes  
+
+# Amplicon library PCR  
+Because of how many reads machines like MiSeq and NextSeq achieve per run, you'll likely want to pool multiple amplicon libraries together in one run when it comes time to sequence, meaning you'll want to use primers that will put different indices on each amplicon library in a given group that you're pooling. As of the writing of this protocol as a markdown file (2020-05-01), all of our lab's oligo pools are designed such that the following primer sequences will fit the purpose of this PCR:  
+
+**Fwd primer:** caagcagaagacggcatacgagatXXXXXXgtgactggagttcagacgtgtgctcttccgatctACTGGCCGCTTCACTG (where XXXXXX is the index; JR4-17 and JR45 in our primer database are all versions of this primer with different indices)  
+**Rev primer:** AATGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTCCGATCT (named JR3 in our primer database)  
```

Comment:
Nice, how did we end up at this one :)? Just a question doens't have to marked down  -- or can

Reply:
End up at the primer sequences? Those primer sequences came from the Harvard MPRNA protocol, so I don't know the details for how the sequences were designed. I presume that the 5' ends of the primers include adaptors required for Illumina sequencing, and the 3' ends of course anneal to sequences included in the oligo pool design, but I don't know how those specific sequences were chosen for the oligo pool design (maybe @taeyoungh can answer that). As for the indices in JR4-17 and JR45, those are all reverse compliments of TruSeq indices (why they took the reverse complement, I'm not sure).